MzML/Mzml.cs

```diff
@@ -354,6 +354,24 @@ private static MsDataScan GetMsDataOneBasedScanFromConnection(Generated.mzMLType
                 }
             }
 
+            double high = double.NaN;
+            double low = double.NaN;
+
+            if (_mzMLConnection.run.spectrumList.spectrum[oneBasedIndex - 1].scanList.scan[0].scanWindowList != null)
+            {
+                foreach (Generated.CVParamType cv in _mzMLConnection.run.spectrumList.spectrum[oneBasedIndex - 1].scanList.scan[0].scanWindowList.scanWindow[0].cvParam)
+                {
+                    if (cv.accession.Equals(_scanWindowLowerLimit))
+                    {
+                        low = double.Parse(cv.value);
+                    }
+                    if (cv.accession.Equals(_scanWindowUpperLimit))
```

Comment:
else if

Reply:
dude. i didn't break this

Reply:
I understand this is just a copy paste, but it's an easy improvement to make to the code since you're working on it right now.

Reply:
done